.drone.yml

```diff
@@ -1,70 +1,63 @@
 kind: pipeline
 name: default
 
-volumes:
-- name: cache
-  temp: {}
-
 steps:
-- name: build
-  image: plugins/docker
-  settings:
-    tags:
-      - "$(DRONE_COMMIT_SHA:0:7)"
-      - latest
-    repo: docker.netsoc.co/public/netsoc.co
-    registry: docker.netsoc.co
-    username:
-      from_secret: docker_dev_username
-    password:
-      from_secret: docker_dev_auth
-  volumes:
-  - name: cache
-    path: /var/lib/docker
-  when:
-    event:
-      - push
-    branch:
-      - new_website
+  - name: build
+    image: plugins/docker
+    settings:
+      tags:
+        - "$(DRONE_COMMIT_SHA:0:7)"
+        - latest
```

Comment:
if we're leaving things on new_website branch for now, will we use `new` tag so it doesnt overwrite current website latest tag? We'll still have a commit-hash tagged image for the old website, but atm things point to latest

Reply:
`new` should be grand